Lib/test/test_concurrent_futures/test_init.py

```diff
@@ -20,6 +20,10 @@
 def init(x):
     global INITIALIZER_STATUS
     INITIALIZER_STATUS = x
+    # InterpreterPoolInitializerTest.test_initializer fails
+    # if we don't have a LOAD_GLOBAL.  (It could be any global.)
+    # We will address this separately.
+    INITIALIZER_STATUS
```

Comment:
@markshannon, any ideas on why this is happening?  It smells like a ceval bug, but it certainly could be something I've done wrong.